acymailing.com.acymailer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "providerId": "acymailing.com",
+    "providerName": "AcyMailer",
+    "serviceId": "acymailer",
+    "serviceName": "AcyMailer DKIM Authentication",
+    "version": 1,
+    "logoUrl": "https://www.acymailing.com/wp-content/uploads/2019/05/ACYMAILING_logo_mobile.png",
+    "description": "Allows user to easily set up CNAME record for AcyMailer service",
+    "syncPubKeyDomain": "acymailer.com",
+    "syncRedirectDomain": "api.acymailer.com, acymailer.com",
+    "variableDescription": "mappingValue1, mappingValue2 and mappingValue3 are the values assigned to host websiteValue1, websiteValue2 and websiteValue3",
+    "sharedProviderName": true,
+    "syncBlock": true,
+    "records": [
+        {
+            "type": "CNAME",
+            "host": "%websiteValue1%",
```

Comment:
if this is for DKIM it would be better to have hostnames like "%dkimhost1%._domainkey" to limit the scope. Fully open and generic CNAMEs pose always a security risk so better avoid them if not necessary.

Reply:
I've changed it to include the `._domainkey` and I've also changed the `pointsTo` to only have the value